app/views/shared/_nav.html.erb

```diff
@@ -0,0 +1,5 @@
+<nav>
+   <div class="nav-wrapper orange darken-3">
```

Comment:
@CjMoore I'm going to start an application style sheet with just a skeleton and the start of a color glossary. There are some cool color palette generators, I can use orange darken-3 as a launching off point and share some options with the group? Let me know if that sounds good.